### DISCUSSION TITLE
Rate change mods affect break time count speed

### DISCUSSION THREAD

**Discussion by DarnTwine** [ppy/osu#25255] [Q&A] [answered]
I don't know if this is intended or not, but rate change mods currently affect the counting speed during break times. Should I report this as an issue? It doesn't seem right to me that a rate change mod should redefine what a second means.

0.5x speed:

https://github.com/ppy/osu/assets/149099574/c6a5ee0e-f1e9-428f-8957-8675c64093df

1x speed:

https://github.com/ppy/osu/assets/149099574/cce7c83e-f2dc-4ddc-ba42-db4c0c4f7f9f

2x speed:

https://github.com/ppy/osu/assets/149099574/3695b8b9-db7f-48f0-abff-f079cc932af8

**Comment by bdach** [comment]
They also change the speed of progression of the song progress bar on skins that have it.

I would argue that the current behaviour is more correct. See: any media player that when told to speed up, doesn't correct the runtime of the media, but speeds up the rate of progress of the progress display(s).

**Reply by bdach** [reply, marked as answer]
See also: discussion in https://github.com/ppy/osu/issues/8929#issuecomment-623014048
